src/main/java/uk/gov/pay/ledger/transaction/model/Payment.java

```diff
@@ -85,6 +85,7 @@ public Payment(Long id, String gatewayAccountId, Long amount,
         this.live = live;
         this.source = source;
         this.walletType = walletType;
+        this.gatewayTransactionId = gatewayTransactionId;
```

Comment:
I think this was intending to set the newly added payout id
```suggestion
        this.gatewayPayoutId = gatewayPayoutId;
```

Reply:
ah dunno why i did this, it's set in the base class constructor so this line just needs to go

Reply:
fixed

Reply:
Yeah much simpler 👍 looks good